recipes/riboseed/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "riboSeed" %}
+{% set version = "0.4.16" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "abb680a514058c2233f34a3c574727f61ed65e06b2123964b359683ccf9eb9f5" %}
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://test-files.pythonhosted.org/packages/21/59/ef9dbb631f132c312bdd1cc145395050fdd0bfbe46765280c9a157367dcd/riboSeed-0.4.16.tar.gz
```

Comment:
Why not using this URL? https://github.com/nickp60/riboSeed/archive/0.4.13.tar.gz

Reply:
Hi @bgruening , thanks for the comments!  This is a alpha test of getting all the dependencies working.  This is why I haven't linked to the stable Pypi link yet.